### ISSUE TITLE
Bug: Apostrophes in text field breaks the query

### ISSUE THREAD

**Issue by klk-amboss** [metabase/metabase#8708]
I am using:
Browser: Chrome 68.1
Operating system: OS
Databases: BigQuery
Metabase version: 0.30.2
Metabase hosting environment: Google App Engine
Metabase internal database: Google SQL

When the field has apostrophes, the Custom Query does not work due to the following syntax:

SELECT university_name 
FROM table
WHERE university_name = 'Taylor's University'

![screen shot 2018-10-17 at 1 24 02 pm](https://user-images.githubusercontent.com/39829548/47083133-07626a00-d210-11e8-8975-92ed039345ce.png)

Thanks for Metabase, amazing tool :)

**Comment by mazameli** [comment]
I think all you have to do is escape that apostrophe using an additional one, like this:

```
SELECT university_name
FROM table
WHERE university_name = 'Taylor''s University'
```

Does this resolve your issue? Pretty sure this is not a bug.

**Comment by nathan-fps** [comment]
You can do this in the SQL editor, but the Custom Query builder should be generating a query that escapes this apostrophe for you.

**Comment by mazameli** [comment]
Wow yeah, sorry, I clearly missed the part in your description where you showed that this was a custom GUI query rather than a SQL one.

**Comment by nathan-fps** [comment]
The TODO statement below points the the likely culprit:

https://github.com/metabase/metabase/blob/14dd7d8b3ebf482e2fe2f004fdec038c29d13808/modules/drivers/bigquery/src/metabase/driver/bigquery.clj#L371-L376

Just need to add some escaping logic there

**Comment by devilelephant** [comment]
Just to note, different database engines have slightly different escaping logic.   BigQuery requires a `\"` where other databases use `''`.  I'm glad this got raised in priority today.

**Comment by Rohith6215** [comment]
We still see this issue this BigQuery data connector the single quotes are still a problem in the version 0.33.2. Is their any workaround (or) can we expect this in the next release.

![image](https://user-images.githubusercontent.com/49909567/65375172-0a7f9900-dcb0-11e9-824f-9bf1acdf793a.png)

**Comment by flamber** [comment]
Fixed in 0.35.4
